tests/test_instrument_tracing.py

```diff
@@ -0,0 +1,87 @@
+import os
+from collections import deque
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from unittest import mock
+
+from opentelemetry.sdk.trace.export import SpanExportResult
+
+from pubtools.tracing import TracingWrapper, instrument_func
+
+
+def test_instrument_func_multiple_threads():
+    trace_id = "cefb2b8db35d5f3c0dfdf79d5aab1451"
+    span_id = "1f2bb7927f140744"
+    os.environ["PUB_OTEL_TRACING"] = "true"
```

Comment:
If you change os.environ directly, it's not reverted at the end of the test, so this test might influence later tests run by the same process.

Please use pytest monkeypatch fixture instead.  If you `monkeypatch.setenv`, whatever you set is automatically rolled back at the end of the test.

Reply:
Updated those tests to use monkeypatch